netcore/Makefile.am

```diff
@@ -56,7 +56,7 @@ xtest-%: prepare check-env dl-test-assets
 	echo -n "***************** $* *********************"
 	ln -sf $(CURDIR)/sdk/$(NETCORESDK_VERSION)/System.Text.Encoding.CodePages.dll assets/extracted/$*/
 	cp $(COREFX_BINDIR)/runtime/netcoreapp-OSX-Debug-x64/xunit.console.deps.json assets/extracted/$*/
-	cp $(COREFX_BINDIR)/runtime/netcoreapp-OSX-Debug-x64/{CoreFx.Private.TestUtilities.dll,Microsoft.DotNet.PlatformAbstractions.dll} assets/extracted/$*/
+	cp $(COREFX_BINDIR)/runtime/netcoreapp-OSX-Debug-x64/CoreFx.Private.TestUtilities.dll assets/extracted/$*/
```

Comment:
Reverts change from https://github.com/mono/mono/pull/13776#discussion_r271261073

Reply:
yeah more recent CoreFX doesn't have this file there (so the rule fails with file not found)